exopy_hqc_legacy/instruments/drivers/visa/agilent_psg_signal_generators.py

```diff
@@ -150,3 +151,41 @@ def output(self, value):
             mess = fill(cleandoc('''The invalid value {} was sent to
                         switch_on_off method''').format(value), 80)
             raise VisaTypeError(mess)
+
+    @instrument_property
+    @secure_communication()
+    def phase(self):
+        """Phase getter method
+        """
+        phase = self.query(':PHASe?')
+        if phase:
+            return float(phase)
+        else:
+            raise InstrIOError
+
+    @phase.setter
+    @secure_communication()
+    def phase(self, value):
+        """Phase setter method
+        """
+        pi = 3.141592653589793
```

Comment:
You can get it either from the math module or from numpy, it will be cleaner.

Reply:
you could by the way use numpy functions to do the conversion deg2rad, rad2deg

Reply:
I was afraid to add numpy at the begining of the script haha
Thanks for the tip

Reply:
You could at least get pi from math... if you do not want to bring numpy in